classes/patterns/class-highlightedcta.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Highlighted CTA pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Class Highlighted CTA.
+ *
+ * @package P4GBKS\Patterns
+ */
+class HighlightedCta extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/highlighted-cta';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 */
+	public static function get_config(): array {
+		return [
+			'title'      => __( 'Highlighted CTA', 'planet4-blocks-backend' ),
+			'categories' => [ 'planet4' ],
+			'content'    => '
```

Comment:
It's probably nicer if we create a function that returns only content, since these patterns get quite complex, defining them inside a key of an array is kind of inconvenient.
Maybe it's even better to put the content in a `.html` file :thinking: